docs(how-tos): add extra section for transferring data block storage MTA-1357 (#79)
scaleway/docs-content: storage/block/how-to/mount-and-use-volume.mdx

## How to format the Block Volume

1. Create a file system with the following command. This command uses the `ext4` filesystem, though you can choose another if you prefer.

  ```
  mkfs.ext4 /dev/sda
  ```

2. Make sure your file system is correctly created by running the `lsblk -f` command.

  ```
  lsblk -f
  ```

  You should see an output like the following. Check that the `FSTYPE` field matches `ext4` for your Block volume.

  ```sh no-copy
  root@scw-festive-agnesi:~# lsblk -f
  NAME    FSTYPE LABEL UUID                                 MOUNTPOINT
  sda     ext4         925c2c17-ca8c-493a-a9cd-b475d87fd276
  vda
  ├─vda1  ext4   root  8509fe04-d91c-410a-9087-c5d34537d3ae /
  └─vda15 vfat         B3E7-7040                            /boot/efi
  ```

## How to mount the Block Volume

Once your file system is created, you need to mount it.  

1. Create the mount point. In this tutorial, we choose `/mnt/block-volume`.

  ```
  mkdir /mnt/block-volume
  ```

> 2. Mount the volume. It is recommended to use the `defaults` option, as in the command below.

Reviewer: I know we're supposed to avoid using "we", but we're also supposed to avoid passive voice, so I'll let you decide which one is best hehe
```suggestion
2. Mount the volume. We recommend you use the `defaults` option, as in the command below.
```

```

## How to transfer data from your local machine to the remote block volume

You may wish to transfer files from your local machine to your Instance's remote block volume. This can be simply achieved with [rsync](https://en.wikipedia.org/wiki/Rsync), a tool for efficiently transferring and copying files. The rsync utility is pre-installed on most Linux distributions and macOS. 

1. Check that rsync is installed on your local machine with the following command:

  ```
  rsync --version
  ```

  You should see an output similar to the following:

    ```sh no-copy
    rsync  version 3.1.3  protocol version 31
  Copyright (C) 1996-2018 by Andrew Tridgell, Wayne Davison, and others.
  Web site: http://rsync.samba.org/
  Capabilities:
      64-bit files, 64-bit inums, 64-bit timestamps, 64-bit long ints,
      socketpairs, hardlinks, symlinks, IPv6, batchfiles, inplace,
      append, ACLs, xattrs, iconv, symtimes, prealloc

  rsync comes with ABSOLUTELY NO WARRANTY.  This is free software, and you
  are welcome to redistribute it under certain conditions.  See the GNU
  General Public Licence for details.
  ```

  <Message type="tip">

>   If you get a `command not found` output (or similar), you need to install rsync. On Linux Ubuntu and Debian systems, you can do this with the APT package tool using the command `sudo apt install rsync `. For Linux CentOS/ Fedora, use the YUM package manager: `sudo yum install rsync`. On Mac OSX with the Homebrew package manager, use `brew install rsync`.

Reviewer: ```suggestion
  If you get a `command not found` output (or similar), you need to install rsync. On Linux Ubuntu and Debian systems, you can do this with the APT package tool using the command `sudo apt install rsync`. For Linux CentOS/ Fedora, use the YUM package manager: `sudo yum install rsync`. On Mac OSX with the Homebrew package manager, use `brew install rsync`.
```
